kuma/users/views.py

```diff
@@ -289,6 +295,17 @@ def revision_by_distinct_doc(list_of_revisions):
     return [documents[doc_id] for doc_id in sorted(documents)]
 
 
+def ban_and_revert_notification(spammer, moderator, info):
+    subject = u'[MDN] %s has been banned by %s' % (spammer, moderator)
+    context = {'spammer': spammer,
+               'moderator': moderator}
+    context.update(info)
```

Comment:
Not a good Idea to change the context later on with info. In future if we can use this function for other view or change any to the passed context, so it will raise issue.
This thing can be done like
```python
def ban_and_revert_notification(spammer, moderator, context):
    subject = u'[MDN] %s has been banned by %s' % (spammer, moderator)
    context['spammer'] = spammer
    context['moderator'] = moderator
```